var/spack/repos/builtin/packages/regtools/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Regtools(CMakePackage):
+    """Tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory
+    and splicing context"""
+
+    homepage = "https://regtools.readthedocs.org/"
+    url = "https://github.com/griffithlab/regtools/archive/refs/tags/1.0.0.tar.gz"
+
+    license("MIT", checked_by="A-N-Other")
+
+    version("1.0.0", sha256="ed2b9db6b71b943924002653caee18511a22ed7cc3c88f428e7e9e0c2e4f431b")
+
+    depends_on("cmake@2.8.12:", type="build")
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
+        with working_dir(self.build_directory):
+            install("regtools", prefix.bin)
```

Comment:
Can you use the `install_tree` function?

Reply:
No, there's a lot of other stuff in that build directory, so I don't want to be transferring rubbish.

Reply:
What is wrong with:
```python
install_tree(os.path.join(self.build_directory, "regtools"), prefix.bin)
```
?

Reply:
Sure, if you prefer. I'd always seen `install_tree` used to copy a whole source directory across rather than just to move across a binary - I'd assume looking at that code that `regtools` was a dir that I was wanting renamed to `bin`.

Reply:
Oh, my bad then - I exactly assumed that was a dir. Then the only request is to copy without cd'ing into the directory. Otherwise LGTM.

Reply:
👍 As requested